src/hub.html

```diff
@@ -24,7 +24,7 @@
     </audio>
 
     <a-scene
-        renderer="antialias: true"
+        renderer="antialias: true; gammaOutput: true; sortObjects: true; physicallyCorrectLights: true;"
```

Comment:
Any perf costs to these? Particularly sortObjects and physicallyCorrectLights?

Reply:
We're already using `sortObjects` I just moved the setting of that renderer option here.

I believe `physicallyCorrectLights` should have a negligible change in perf. The shader changes a little, but I don't think it's too much more expensive.